frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -134,17 +135,23 @@ function convertUniversalFiltersToLegacyFilters(universalFilters: RecordingUnive
         } else if (isActionFilter(f)) {
             actions.push(f)
         } else if (isAnyPropertyfilter(f)) {
-            properties.push(f)
+            if (f.type === PropertyFilterType.Recording) {
+                // TODO: add console log filtering
```

Comment:
I believe this branch will never be executed so we don't need to show an unsupported message or something?

Reply:
Doesn't get called right now but will be used in a follow up PR which is up https://github.com/PostHog/posthog/pull/22869